src/scripts/main.js

```diff
@@ -357,4 +357,23 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('table.dashboard');
```

Comment:
Ensure that there is a table element with the class 'dashboard' in your HTML file. Otherwise, the script will not be able to append rows to it.